HIP/hip-1084.md

```diff
@@ -0,0 +1,111 @@
+network.
+
+## Rationale
+
+On Ethereum, all execution costs are paid for by the 'sender' who created, signed, and submitted the transaction. Hedera
+follows a similar approach for all `ContractCallTransaction`s but also charges an additional fee for each
+`EthereumTransaction` submitted via JSON-RPC relays. This fee serves as a defense against a malicious JSON-RPC relay that
+might send malformed transactions to the network, where there would be no one to pay for the processing of those transactions.
+This fee is unnecessary for successful transactions since the intrinsic gas fee covers the base costs (gossip & consensus).
+Therefore, this HIP proposes that the fee for submitting `EthereumTransactions` to the network should only be charged if the
+JSON-RPC relay submits a transaction that fails due diligence checks, such that the EOA cannot be charged for the overhead
+of gossip and consensus.
+
+## User stories
+
+1. As a relay operator, I want to submit an `EthereumTransaction` on behalf of an EOA without paying additional fees.
+2. As a node operator, I want to charge relays that submit transactions that cannot be charged to the EOA.
+3. As a user, I want to use the same pricing structure and have the same user experience for Hedera relays as with other
+   networks.
+  
+## Specification
+
+Definitions:
+ - `evm tx`: The RLP encoded transaction bytes defined by the Ethereum yellow paper that form a standard Ethereum transaction
+ - `relay`: A system that, given an `evm tx`, wraps it in an HAPI `EthereumTransaction` and submits it to Hedera.
+            This is also the `payer` for the `EthereumTransaction`. See
+            [TransactionBody.transactionID](https://github.com/hashgraph/hedera-protobufs/blob/main/services/transaction_body.proto#L104)
+ - `EOA`: The account that created and signed the `evm tx` and submitted it to a `relay`. This is the `from` in the
+          [Ethereum Transaction Schema](https://github.com/ethereum/execution-apis/blob/main/src/eth/transaction.yaml)
+          often noted as (tx.from)
+
+### Smart Contract Creation and Execution Transactions
+
+The `relay` will **only** be charged if handling of the `EthereumTransaction` fails **before** any bytecodes are executed.
+In other words, the `relay` is charged when one of the pre-requisite checks for `EthereumTransaction` fails, regardless of
+the execution status of the transaction itself once it begins smart contract creation or execution in the EVM.
+
+Examples:
+ - The `relay` is charged if the `evm tx` cannot be parsed or is otherwise malformed
+ - The `relay` is charged if the `EOA` specified in the `evm tx` refers to a non-existent account
+ - The `relay` is charged if the `evm tx` was not properly signed by the keys of the `EOA`
+ - The `relay` is charged if the `EOA` has insufficient funds to pay the intrinsic gas fee
+ - The `relay` is charged if the `nonce` check fails
```

Comment:
This indeed highlights a list of prechecks performed by the Relay to validate transactions and reject them if any conditions are met, ensuring they are not forwarded to the network unnecessarily, thus avoiding charges. Notably, the Hedera JSON-RPC Relay already handles and rejects these scenarios. However, there are additional prechecks currently performed by the Relay that are not mentioned here:

- Transaction type  
- Gas price  
- Gas limit  
- Chain ID  
- Value  

I recommend considering these prechecks and including them in this HIP for greater clarity and completeness.

Reply:
I agree, thank you @quiet-node! Can you please reply to this comment with a suggested replacement (you can click the icon of a document with the + and - signs within it to offer a suggestion) and we will get it applied to the HIP text.

Reply:
If a precheck is performed by the Relay, doesn't this mean that if it fails, the Relay will never send a transaction to a Hedera node, meaning the relay account can never be charged for it?

Reply:
Correct. 
A relay will only submit to a CN a transaction that passes it's prechecks.
If it fails it is in the best interest of the relay to not submit to a CN otherwise it will be charged.